stable/postgresql/templates/deployment.yaml

```diff
@@ -32,29 +32,35 @@ spec:
             secretKeyRef:
               name: {{ template "fullname" . }}
               key: postgres-password
+        - name: POD_IP
+          valueFrom: { fieldRef: { fieldPath: status.podIP } }
         ports:
         - name: postgresql
           containerPort: 5432
         livenessProbe:
           exec:
             command:
-            - pg_isready
+            - sh
+            - -c
+            - exec pg_isready --host $POD_IP
           initialDelaySeconds: 60
           timeoutSeconds: 3
         readinessProbe:
           exec:
             command:
-            - pg_isready
-          initialDelaySeconds: 60
-          timeoutSeconds: 3
+            - sh
+            - -c
+            - exec pg_isready --host $POD_IP
+          initialDelaySeconds: 5
+          timeoutSeconds: 1
```

Comment:
I'm a little worried that this makes the postgres availability flaky in high load situations.
What do you think about relaxing the timeouts and maybe increase the `failureThreshold` on the livenessProbes instead.